README.md

```diff
@@ -10,192 +10,9 @@ The code is documented with javadoc and therefore should be usable from viewing
-        .environment("development")
-        .build();
-Rollbar rollbar = Rollbar.init(config);
-rollbar.error(t);
-```
-
-There is a shim that has the same basic API as the old library located in the `rollbar-java`
-package at `com.rollbar.Rollbar`. This class is marked as deprecated as it is only intended to
-make upgrading slightly more convenient. This old example code should still work thanks to this shim class:
-
-```java
-import com.rollbar.Rollbar;
-public class MainClass {
-    public static final Rollbar rollbar = new Rollbar("ACCESS_TOKEN", "production");
-    public int main(String[] args) {
-        rollbar.handleUncaughtErrors();
-        OtherClass.runProgram();
-        return 0;
-    }
-}
-```
-
-However, we strongly advise upgrading to at least this equivalent using the new library:
-
-```java
- import com.rollbar.notifier.Rollbar;
- public class MainClass {
-     public static final Rollbar rollbar = new Rollbar(
-         withAccessToken("ACCESS_TOKEN")
-             .environment("production")
-             .handleUncaughtErrors(true)
-             .build());
-     public int main(String[] args) {
-         OtherClass.runProgram();
-         return 0;
-     }
- }
-```
-
-## Installing
-
-You can, of course, build it yourself and depend on the .jar manually,
-however, the modules are up on maven central and can be installed in
-most tool chains.
-
-### Maven
-
-All these can be installed as Maven projects. Simply add the
-dependency to your pom file:
-
-```xml
-<dependencies>
-<dependency>
-  <groupId>com.rollbar</groupId>
-   <artifactId>rollbar-java</artifactId>
-   <version>1.2.0</version>
-</dependency>
-</dependencies>
-```
-
-### Gradle
-
-```groovy
-compile('com.rollbar:rollbar-java:1.2.0')
-```
-
-### Android
-
-As described above, this library is split into different components that build upon each other.
-There is an Android specific part of the library. Therefore for Android you should use that
-interface which requires you to add this dependency to your build process. For example,
-
-```groovy
-compile('com.rollbar:rollbar-api:1.2.0')
-compile('com.rollbar:rollbar-java:1.2.0')
-compile('com.rollbar:rollbar-android:1.2.0@aar')
-```
-
-## How payloads are sent
-
-The actual notifier configuration builds a notifier that uses a BufferedSender to send the items
-to Rollbar. That sender is built using an unbound memory queue and a scheduled thread to send
-the events from the queue. 
-The queue as well as the frequency of the scheduled thread can be customized
-when building the buffered sender and it can be replaced by passing a custom configuration when 
-creating the notifier or initializing it.
+## Setup Instructions
+1. [Sign up for a Rollbar account](https://rollbar.com/signup)
+2. Follow the [Installation](https://docs.rollbar.com/v1.0.0/docs/java#section-installation) instructions in our [Java SDK docs](https://docs.rollbar.com/v1.0.0/docs/java) to install rollbar-java and configure it for your platform.
```

Comment:
What does the `v1.0.0` in the url actually specify? In those docs it is referring to v1.2.0 which is the latest.